crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -449,6 +449,7 @@ pub struct CheckpointSignatureAggregator {
     summary: CheckpointSummary,
     digest: CheckpointDigest,
     signatures: StakeAggregator<AuthoritySignInfo, true>,
+    failures: StakeAggregator<AuthoritySignInfo, false>,
```

Comment:
nice! love how we can leverage the `StakeAggregator` for two different purposes with just a difference of `false` vs `true`